app/views/step_by_step_pages/_form.html.erb

```diff
@@ -6,7 +6,10 @@
 <%= form_for(@step_by_step_page) do |form| %>
   <% if step_by_step_page.errors.any? %>
     <% error_explanation = capture do %>
-      <h2 class="h4">There are <%= pluralize(step_by_step_page.errors.count, "error") %></h2>
+      <h2 class="h4">
+        There <%= step_by_step_page.errors.count == 1 ? "is" : "are" %>
```

Comment:
You can extract this to a helper method and use it in both places where you need to show error messages

Reply:
Do you think it's worth it extracting just that line?
I was thinking of rather creating a partial for the whole snippet to reduce duplication.

Reply:
In which helper would that need to be? Because it is used both for `step_by_step_pages` as well as `steps`. Should it be in the `application_helper`?
I wonder if it makes more sense to rewrite the heading so that wouldn't be necessary... ;-)

Reply:
I have now fixed this by rewriting the heading, but also moved everything into a partial.
We never use headings as specific as "There are 2 errors" elsewhere. Our guidance on theses error summary headings is not very clear but something like "There is a problem" (to which I have changed this heading) is most commonly used, as far as I know.